packages/ERTP/test/unitTests/test-mintObj.js

```diff
@@ -6,6 +6,7 @@ import { assert } from '@agoric/assert';
 
 import { defineDurableKind, makeKindHandle } from '@agoric/vat-data';
 import { makeIssuerKit, AssetKind, AmountMath } from '../../src/index.js';
```

Comment:
The refactored tests in ERTP all verify that the old behavior continues to work. There should be tests that show that the recovery purses have the ability to recall the payment and that the mint (or separate authority facet) has access to the recovery set.